src/js/workshop-comments.js

```diff
@@ -0,0 +1,144 @@
+const commentForm = document.getElementById("comment-form");
+
+const nameDOMElement = document.getElementById("name");
```

Comment:
Line 3 & 4 are duplicates of line 9 & 11. It would be better to remove these lines of global scoped variables but rather using the line 9 & 11 that are functional scoped.

As to the use of these variables in `setSubmittedComment()` function, improve this function by accepting two parameters of `name` and `comment` values generated at line 10 & 12 so that `setSubmittedComment()` function doesn't need to re-calculate these values.